colaraz/lms/static/js/discussion/colaraz_discussion_utils.js

```diff
@@ -0,0 +1,57 @@
+$(document).on("click", "button.action-vote", function(event) {
+    var currentElement = this;
```

Comment:
shouldn't we use 'let' for such common names?

Reply:
DONE.